android/app/src/main/java/org/fossasia/openevent/core/notifications/NotificationsFragment.java

```diff
@@ -31,6 +31,7 @@
     private NotificationsFragmentViewModel notificationsFragmentViewModel;
     private NotificationsAdapter notificationsAdapter;
     private List<Notification> notificationsList;
+    private boolean downloadResponse = false;
```

Comment:
This will only be triggered once in the whole lifetime of the class. Wrong behaviour